lib/fluent/plugin/out_memcached.rb

```diff
@@ -33,12 +31,21 @@ def start
 
   def shutdown
     @memcached.close
+    super
   end
 
   def format(tag, time, record)
     [tag, time, record].to_msgpack
```

Comment:
Thank you for the example! Now I understand. It's better to reduce the chunk size.
Just for my curiosity, what kind of cases should the chunk include tag? (It means `[tag, time, record].to_msgpack`, not `[time, record].to_msgpack`)